packages/react-ui-rent/src/LeadForm/Fields/RequiredField.js

```diff
@@ -44,6 +45,10 @@ export default class RequiredField extends Component {
     } else {
       this.setState({ labelVisible: true })
     }
+    if (this.props.onChange &&
+      typeof this.props.onChange === 'function') {
```

Comment:
probably don't need the typeof check, as you have the prop type validation above